pages/content/amp-dev/documentation/guides-and-tutorials/start/create_successful_stories.md

```diff
@@ -1,23 +1,24 @@
 ---
-$title: Successfully create stories on the web
+$title: Successfully create Web Stories
 $order: 1
-description: 'Follow these best practices to create a satisfying snackable story'
+description: 'Follow these best practices to create a satisfying snackable web story'
```

Comment:
for my own edification, is it ever `Web Story`? Or is there any kind of guide as to when to use the capitalized version?

Reply:
From my understanding, only in titles - it isn't a proper noun